tools/test-tools/CHANGELOG.md

```diff
@@ -2,10 +2,10 @@
 
 ## vNext
 
-Dependency updates.
-
 ## 2.0.0
 
 ### ⚠ BREAKING CHANGES
 
-Update `typescript` dependency from `4.x` to `5.x`.
+- Dependency updates.
```

Comment:
This bullet isn't a breaking change. Presumably it should go before the breaking changes heading.